run_service.sh

```diff
@@ -69,4 +108,4 @@ docker rm -f abci0 node0 trader_abci_0 trader_tm_0 &> /dev/null ||
 
 # Install dependencies and run the agent througth the middleware
 poetry install --only main --no-cache
-poetry run python -m operate.cli quickstart "$1"
+poetry run python -m operate.cli quickstart "$CONFIG_PATH" $ATTENDED_ARG
```

Comment:
why not
```suggestion
poetry run python -m operate.cli quickstart $@
```
and handle the rest in middleware?

Reply:
updated as per suggestions